python/doc/theory/reliability_sensitivity/importance_form.rst

```diff
@@ -70,13 +71,13 @@ Importance factors from FORM method
    importance factors ... (see [lebrun2009c]_ to compare
    the different isoprobabilistic transformations).
 
-| It is possible to give another definition to the importance factors
-  which may be defined in the elliptical space of the iso-probabilistic
-  transformation, where the marginal distributions are all elliptical,
-  with cumulative distribution function noted :math:`E`, and not yet
-  uncorrelated.
+It is possible to give another definition to the importance factors
+which may be defined in the elliptical space of the iso-probabilistic
+transformation, where the marginal distributions are all elliptical,
+with cumulative distribution function noted :math:`E`, and not yet
+uncorrelated. We denote by :math:`F_i` the cumulative distribution of the component :math:`i`.
```

Comment:
I would add a link to the "Isoprobabilistic transformation" page here, because it defines what exactly "E" means.

Reply:
ok.